src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -170,11 +171,51 @@ internal static bool IsSupportedRsaAlgorithm(string algorithm)
                 case SecurityAlgorithms.RsaPKCS1:
                 case SecurityAlgorithms.RsaOaepKeyWrap:
                     return true;
+                case SecurityAlgorithms.RsaSsaPssSha256:
+                case SecurityAlgorithms.RsaSsaPssSha384:
+                case SecurityAlgorithms.RsaSsaPssSha512:
+                case SecurityAlgorithms.RsaSsaPssSha256Signature:
+                case SecurityAlgorithms.RsaSsaPssSha384Signature:
+                case SecurityAlgorithms.RsaSsaPssSha512Signature:
+                    return IsSupportedRsaPss(key);
             }
 
             return false;
         }
 
+        private static bool IsSupportedRsaPss(SecurityKey key)
+        {
+#if NET45 || NET451
```

Comment:
If the application is targeting NET451, but NET48 runtime is installed, RsaPSS will actually be supported. We need to expand our work 'lightup' work here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/ce23b8c4002d168d6ab2f9d971a9993e5cba99cc/src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs#L515

Reply:
I assumed that users running .NET 4.8 would use our .NET 4.6.1 target, but I see use cases where this could be important e.g. Katana supports only .NET 4.5 target.

Reply:
Katana's target doesn't matter, nuget resolves based on the application target.

Reply:
@GeoK we should open a new issue for this.

Reply:
Oh, that's right. Thanks @Tratcher.
@brentschmaltz In what scenarios would users targeting e.g. .NET 4.6.1 have to bind to our .NET 4.5 target?